studio/deskStructure.js

```diff
@@ -27,10 +23,32 @@ export default () =>
         )
         .icon(MdHome),
       S.listItem()
-        .title('Pages')
-        .schemaType('page')
-        .child(S.documentTypeList('page').title('Page'))
-        .icon(MdLayers),
+        .title('Services Page')
+        .schemaType('servicesPage')
+        .child(
+          S.editor()
+          .id('servicesPage')
+          .schemaType('servicesPage')
+          .documentId('servicesPage'))
+        .icon(MdAssignment),
```

Comment:
Should probably normalize this. «Services Page» seems specific, it should probably be `services` (Services).